<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tch_manager_detail</class>
 <widget class="QMainWindow" name="tch_manager_detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AtQlas Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>562</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/tchm.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="stu_table">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>381</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget 
{background-color: transparent;
}
QScrollBar:vertical {
    background: url(images/pure_red.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_red.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: PT Mono;
font-size:12px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="credit_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
      </font>
     </property>
     <property name="text">
      <string>credit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Credit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Grade</string>
     </property>
    </widget>
    <widget class="QLabel" name="grade_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
      </font>
     </property>
     <property name="text">
      <string>grade</string>
     </property>
    </widget>
    <widget class="QLabel" name="optp_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>290</y>
       <width>121</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Optional? </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Optional?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
      </font>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="lid_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
      </font>
     </property>
     <property name="text">
      <string>LID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>420</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/refresh.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
